src/node.h

```diff
@@ -416,7 +416,11 @@ enum Flags : uint64_t {
   kNoRegisterESMLoader = 1 << 3,
   // Set this flag to make Node.js track "raw" file descriptors, i.e. managed
   // by fs.open() and fs.close(), and close them during FreeEnvironment().
-  kTrackUnmanagedFds = 1 << 4
+  kTrackUnmanagedFds = 1 << 4,
+  // Controls whether or not the Environment should call InitializeInspector.
+  // This control is needed by embedders who wish to manage the InspectorAgent
+  // themselves.
```

Comment:
I know it’s probably a large task, but I think the documentation here might benefit from being expanded a bit. In particular, neither `InitializeInspector` nor `inspector::Agent` are public APIs, so referencing them probably isn’t ideal here, and it might be good to explain what exactly it is that embedders (may) need to do when they specify this flag